storage/api/src/test/java/org/apache/kafka/server/log/remote/storage/NoOpRemoteLogMetadataManager.java

```diff
@@ -74,6 +74,11 @@ public void onPartitionLeadershipChanges(Set<TopicIdPartition> leaderPartitions,
     public void onStopPartitions(Set<TopicIdPartition> partitions) {
     }
 
+    @Override
+    public long remoteLogSize(TopicIdPartition topicPartition, int leaderEpoch) {
```

Comment:
can we use same variable name?

topicPartition -> topicIdPartition

Reply:
Great catch! Thank you!